nilearn/decomposition/tests/test_multi_pca.py

```diff
@@ -4,8 +4,8 @@
 
 import numpy as np
 from nose.tools import assert_raises
-
 import nibabel
+from numpy.testing import assert_almost_equal
```

Comment:
Are you sure that this import is useful? It seems that the only change that you have done to the file is to add it.